test/helpers/kubectl.go

```diff
@@ -476,19 +476,23 @@ func (kub *Kubectl) WaitForKubeDNSEntry(name string) error {
 	}
 	// https://bugs.launchpad.net/ubuntu/+source/bind9/+bug/854705
 	digCMD := "dig +short %s @%s | grep -v -e '^$'"
+	digCMDFallback := "dig +tcp %s @%s"
```

Comment:
A comment on what the +tcp does might be good. Most people will be a little surprised ;) Also, perhaps, explain what the fallback does differently. I see we do that where we use it so maybe it's ok, but mention that it prints the whole response.

Reply:
A comment was added when the fallback is in use. Should I move it?